src/models.jl

```diff
@@ -71,7 +73,7 @@ function ntuple_preds(func::F, target_table, idxsvec, weights) where {F <: Funct
 end
 
 function univariate_table(::Type{T}, weights, target_table, idxsvec) where {T}
-    table = if T <: DictTable
+    table = if T <: DictColumnTable
```

Comment:
This is fine, but maybe type dispatch instead of `if...else...end` is more usual?